.github/workflows/ci.yml

```diff
@@ -9,12 +12,7 @@ jobs:
     timeout-minutes: 30
     strategy:
       matrix:
-        python-version: ['3.9', '3.10']
-        editable_install_option: ['-e', '']
-        include:
-        #Test for aiida-core 1.X (since aiida-core 2.X does not support python 3.7)
-        - python-version: '3.7'
-          editable_install_option: '-e'
+        python-version: ['3.7', '3.8', '3.9', '3.10', '3.11', '3.12']
```

Comment:
We can ask on the discourse and if no one using aiida-core < 2 with this anymore, maybe can drop the support for 3.7 and 3.8? 
I think if it is going to make a release, it is not a problem to drop the support and versioning it?

Reply:
Let's have this discussion at #72. There I proposed to release the code basically as is (after renaming it) as version 0.1.0 so that existing users can easily switch to the official package. After that all breaking changes and droping support can be done in v0.2.

Reply:
> one using aiida-core < 2 with this anymore, maybe can drop the support for 3.7 and 3.8?

btw: Dropping support for AiiDA v1 should anyway be done in a separate PR, since there will be a lot of code cleanup associated with that.